relay-server/src/actors/outcome.rs

```diff
@@ -166,145 +169,147 @@ pub enum DiscardReason {
     ProcessUnreal,
 }
 
+impl Outcome {
+    /// Returns the name of the outcome as recognized by Sentry.
+    fn name(&self) -> &'static str {
+        match self {
+            Outcome::Accepted => "accepted",
+            Outcome::Filtered(_) => "filtered",
+            Outcome::RateLimited(_) => "rate_limited",
+            Outcome::Invalid(_) => "invalid",
+            Outcome::Abuse => "abuse",
+        }
+    }
+
+    fn to_outcome_id(&self) -> u8 {
+        match self {
+            Outcome::Accepted => 0,
+            Outcome::Filtered(_) => 1,
+            Outcome::RateLimited(_) => 2,
+            Outcome::Invalid(_) => 3,
+            Outcome::Abuse => 4,
+        }
+    }
+
+    fn to_reason(&self) -> Option<&str> {
+        match self {
+            Outcome::Accepted => None,
+            Outcome::Invalid(discard_reason) => Some(discard_reason.name()),
+            Outcome::Filtered(filter_key) => Some(filter_key.name()),
+            Outcome::RateLimited(code_opt) => code_opt.as_ref().map(|code| code.as_str()),
+            Outcome::Abuse => None,
+        }
+    }
+}
+
+impl DiscardReason {
+    pub fn name(self) -> &'static str {
+        match self {
+            DiscardReason::Duplicate => "duplicate",
+            DiscardReason::ProjectId => "project_id",
+            DiscardReason::AuthVersion => "auth_version",
+            DiscardReason::AuthClient => "auth_client",
+            DiscardReason::NoData => "no_data",
+            DiscardReason::TooLarge => "too_large",
+            DiscardReason::DisallowedMethod => "disallowed_method",
+            DiscardReason::ContentType => "content_type",
+            DiscardReason::MultiProjectId => "multi_project_id",
+            DiscardReason::MissingMinidumpUpload => "missing_minidump_upload",
+            DiscardReason::InvalidMinidump => "invalid_minidump",
+            DiscardReason::SecurityReportType => "security_report_type",
+            DiscardReason::SecurityReport => "security_report",
+            DiscardReason::Cors => "cors",
+            DiscardReason::ProcessUnreal => "process_unreal",
+
+            // Relay specific reasons (not present in Sentry)
+            DiscardReason::Payload => "payload",
+            DiscardReason::InvalidJson => "invalid_json",
+            DiscardReason::InvalidMultipart => "invalid_multipart",
+            DiscardReason::InvalidMsgpack => "invalid_msgpack",
+            DiscardReason::InvalidTransaction => "invalid_transaction",
+            DiscardReason::InvalidEnvelope => "invalid_envelope",
+            DiscardReason::ProjectState => "project_state",
+            DiscardReason::DuplicateItem => "duplicate_item",
+            DiscardReason::Internal => "internal",
+        }
+    }
+}
+
+/// The outcome message is serialized as json and placed on the Kafka topic using OutcomePayload
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct OutcomePayload {
```

Comment:
_nit_: Since this is now a message in the public interface of this module and we usually name messages in _active_ language, how would you feel about `TrackRawOutcome`?